src/TuDa.CIMS.Web/Components/AssetList.razor

```diff
@@ -0,0 +1,195 @@
+<h3>Bestandsliste</h3>
+
+@code {
+
+// Parameter Items for asset list
+    [Parameter] public string[] Items { get; set; }
+
+    // TODO: remove the code for the initialization
+    private string[] _examples =
+    {
+        "Trockeneis", "Ethanol", "Natriumhydroxid", "Aceton", "Ammoniumchlorid", "Wasserstoffperoxid",
+        "Kaliumpermanganat", "Messzylinder", "Erlenmeyerkolben", "Reagenzglas"
+    };
+
+    protected override void OnInitialized()
+    {
+        Items = _examples;
+    }
+
+}
+
+
+@* grid with one column for items *@
+<div class="container text-center">
+    <div class="col">
+        @foreach (var item in Items)
+        {
+            <div
+                class="row custom-checkbox-container">
+                <p>@item</p>
+                <label class="custom-checkbox">
+                    <input type="checkbox"
+                           class="form-check-input"
+                           aria-label="...">
+                    <span class="checkmark"> </span>
+                </label>
+            </div>
+        }
+    </div>
+</div>
+
+@* pagination *@
+<nav
+    aria-label="Page navigation example">
+    <ul
+        class="pagination">
+        <li
+            class="page-item disabled">
+            <a class="page-link">
+                🡸
+            </a>
+        </li>
+        <li
+            class="page-item active"
+            aria-current="page">
+            <a class="page-link"
+               href="#">
+                1
+            </a>
+        </li>
+        <li class="page-item">
+            <a class="page-link"
+               href="#">
+                2
+            </a>
+        </li>
+        <li class="page-item">
+            <a class="page-link" href="#">3</a>
+        </li>
+        <li class="page-item">
+            <a class="page-link"
+               href="#">
+                🡺
+            </a>
+        </li>
+    </ul>
+</nav>
+
+<style>
```

Comment:
🤔 Maybe we should talk about the colors. I know in the example here we used a lot of green, but the others did not exactly copy the example and did not use green. So if you want, you could also change the colors to white or something similar.

Reply:
⛏️ Use the BootStrap class `bg-light` class for the background of the items please